src/marlin_stubs/pause/M600.cpp

```diff
@@ -116,10 +119,48 @@ void GcodeSuite::M600() {
 }
 
 void M600_execute(xyz_pos_t park_point, int8_t target_extruder,
-    xyze_float_t resume_point, std::optional<float> unloadLength, std::optional<float> fastLoadLength, std::optional<float> retractLength,
-    pause::Settings::CalledFrom);
+    xyze_float_t resume_point, std::optional<float> unloadLength, std::optional<float> fastLoadLength,
+    std::optional<float> retractLength, std::optional<filament::Colour> filament_colour,
+    std::optional<filament::Type> filament_type, pause::Settings::CalledFrom);
 
 void M600_manual() {
+    char filamenttype[16] = { '\0' };
```

Comment:
I think nothing ever assigns into this one.

Reply:
Exactly. Removed.  Solved with commit [d631509](https://github.com/prusa3d/Prusa-Firmware-Buddy/pull/3581/commits/d6315094ec575102dc032859100f6114a28a341d)